docs/website/content/docs/v0.3/Guides/bootstrapping.md

```diff
@@ -123,38 +123,14 @@ As of Cluster API version 0.3.9, Sidero is included as a default infrastructure
 To install Sidero and the other Talos providers, simply issue:
 
 ```bash
-clusterctl init -b talos -c talos -i sidero
+SIDERO_METADATA_SERVER_HOST_NETWORK=true \
```

Comment:
Indent issues?

Reply:
I think we can indent it this way, can't we? if you prefer other style, I can format it other way

Reply:
I don't care. Just looked like a typo to me at first glance. The subsequent lines are indented and that threw me off.

Reply:
I was trying to make it clear it's a single command, if all of them have same indentation, it might look like multiple commands